man/udevadm.xml

```diff
@@ -382,6 +382,21 @@
             then each matching result is ORed, that is, all children of each specified device are triggered.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--database-nomatch</option></term>
+          <listitem>
+            <para>Trigger devices without a corresponding udev database entry only. This can potentially save a
+            significant amount of time compared to re-triggering all devices in the system and e.g. can be used to
+            optimize boot time.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><option>--database-match</option></term>
+          <listitem>
+            <para>Trigger devices with a corresponding udev database entry only. This is the inverse operation of
+            --database-nomatch.</para>
```

Comment:
inverse operation → inverse match.

also please enclose `--database-nomatch` in `<option>`/`</option>`